config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+# Be sure to restart your server when you modify this file.
+#
+# This file eases your Rails 7.0 framework defaults upgrade.
+#
+# Uncomment each configuration one by one to switch to the new default.
+# Once your application is ready to run with all new defaults, you can remove
+# this file and set the `config.load_defaults` to `7.0`.
```

Comment:
Just confirming that this file should eventually be removed?

Reply:
Yes it should once every new default from Rails 7 will have been enabled.